test/appium/tests/critical/test_fallback.py

```diff
@@ -159,6 +160,63 @@ def _check_message(home_view, index):
+        self.home_1.navigate_back_to_home_view()
+        self.home_2.navigate_back_to_home_view()
+        wallet_1 = self.home_1.wallet_tab.click()
+        wallet_2 = self.home_2.wallet_tab.click()
+        regular_account_name = "New regular account"
+        key_pair_account_name = "Key pair account"
+        key_pair_name = "New key pair"
+
+        wallet_1.just_fyi("Device 1: add a new regular account")
+        wallet_1.add_regular_account(account_name=regular_account_name)
+        account_element = wallet_1.get_account_element(account_name=regular_account_name)
+        wallet_1.close_account_button.click_until_presence_of_element(account_element)
+
+        wallet_1.just_fyi("Device 1: add a new key pair account")
+        account_element.swipe_left_on_element()
+        wallet_1.add_key_pair_account(account_name=key_pair_account_name,
+                                      passphrase=account_to_add['passphrase'],
+                                      key_pair_name=key_pair_name)
+
+        self.home_2.just_fyi("Device 2: check imported accounts are shown before importing key pair")
+        self.home_2.profile_button.click()
+        self.profile_2.profile_wallet_button.click()
+        self.profile_2.key_pairs_and_accounts_button.click()
+        if not self.profile_2.get_missing_key_pair_by_name(key_pair_name=key_pair_name).is_element_displayed():
+            self.errors.append("Key pair is not shown in profile as missing before importing")
+        if not self.profile_2.get_key_pair_account_by_name(account_name=regular_account_name).is_element_displayed():
+            self.errors.append(
+                "Newly added regular account is not shown in profile as on device before importing key pair")
+        self.profile_2.click_system_back_button(times=3)
+
+        wallet_2.just_fyi("Device 2: import key pair")
+        wallet_2.get_account_element(account_name=key_pair_account_name).click()
+        wallet_2.element_by_translation_id("import-key-pair").click()
+        self.sign_in_2.passphrase_edit_box.send_keys(account_to_add['passphrase'])
+        wallet_2.slide_and_confirm_with_password()
+
+        self.home_2.just_fyi("Device 2: check imported accounts are shown in profile as added after importing key pair")
+        self.home_2.profile_button.click()
+        self.profile_2.profile_wallet_button.click()
+        self.profile_2.key_pairs_and_accounts_button.click()
+        if not self.profile_2.get_key_pair_account_by_name(account_name=regular_account_name).is_element_displayed():
+            self.errors.append("Newly added regular account is not shown in profile after importing key pair")
+        if not self.profile_2.get_key_pair_account_by_name(account_name=key_pair_account_name).is_element_displayed():
+            self.errors.append("Key pair account is not shown in profile as on device after importing key pair")
+        self.profile_2.click_system_back_button(times=3)
```

Comment:
Also here: we need to check deriviation path and wallet address for both wallet accounts: regular_account_name  and imported just to be sure they are matches expected.